core/src/main/java/io/confluent/rest/Application.java

```diff
@@ -150,7 +158,7 @@ public Server createServer() throws RestConfigException {
 
     // Configure the servlet container
     ServletContainer servletContainer = new ServletContainer(resourceConfig);
-    FilterHolder servletHolder = new FilterHolder(servletContainer);
+    final FilterHolder servletHolder = new FilterHolder(servletContainer);
```

Comment:
Does this really complain about the `FilterHolder` but not the `ServletContainer` right above it? Or just unrelated cleanup?